libmscore/select.h

```diff
@@ -172,6 +174,12 @@ class Selection {
       bool isList() const              { return _state == SelState::LIST; }
       void setState(SelState s);
 
+      //! NOTE If locked, the selected items should not be changed.
+      void lock(const QString& reason)    { _lockReason = reason; }
+      void unlock(const QString& reason)  { Q_UNUSED(reason); _lockReason.clear(); } // reason for clarity
```

Comment:
Maybe it's better to just omit the `reason` parameter :-)

Reply:
I think not, it’s not obvious what should contain the argument.

Reply:
Then maybe
`void unlock(const QString& /*reason*/) { _lockReason.clear(); }`
?

Reply:
Then the name of the argument in IDE will not be shown.
![image](https://user-images.githubusercontent.com/3818029/77625830-fb262480-6f4c-11ea-903e-b39481425d4c.png)

Should be like that
![image](https://user-images.githubusercontent.com/3818029/77625926-290b6900-6f4d-11ea-8adb-870393b0c994.png)